### DISCUSSION TITLE
File Export Name No Longer Working?

### DISCUSSION THREAD

**Discussion by denysullivan** [AutomaApp/automa#1095] [Q&A]
Just happened to check my workflows today and my files are now downloading as just 'download (#)' instead of the given names in the export data. When I went in the load and exported it manually it had the correct name. Could it be some chrome setting that is preventing the file download naming?

**Comment by Kholid060** [comment]
Try to delete the Insert Data block and add it because it worked when I tested it.

**Reply by denysullivan** [reply]
![image](https://user-images.githubusercontent.com/94069734/212368361-13848d4c-d7a2-4739-9523-6ba599eea6df.png)
I dont have an insert data, just Get Text that is being inserted to tables. Tried deleted export data and it didnt work

**Comment by denysullivan** [comment]
Any other suggestions? the file name is right when I go in the logs and export the data, but not the actual export in the workflow
![image](https://user-images.githubusercontent.com/94069734/212723511-e1af483b-1fa8-48a9-9e22-7583b6b10eb7.png)

**Reply by Kholid060** [reply]
Can you send me the workflow? Because I can't reproduce the issue that you have

**Reply by denysullivan** [reply]
Sure, here it is. As mentioned, from the logs I can properly export the file with the name, its only when the workflow runs that it is called "download"
[SH288.automa.zip](https://github.com/AutomaApp/automa/files/10447432/SH288.automa.zip)

**Comment by denysullivan** [comment]
Getting back to this - it's something about the interaction with my other extension to route downloads into specific folders (I've turned that off and it works like normal now). Thanks for your help.
